website/content/docs/api-gateway/usage/basic-usage.mdx

```diff
@@ -0,0 +1,131 @@
+---
+layout: docs
+page_title: Consul API Gateway Basic Usage
+description:  >-
+  Consul API Gateway Basic Usage
+---
+
+
+# Basic Usage
+
+1. Verify that the [requirements](/docs/api-gateway/consul-api-gateway-install#requirments) have been met.
+1. Verify that the Consul API Gateway CRDs and controller have been installed and applied (see [Installation](/docs/api-gateway/consul-api-gateway-install)).
+1. Configure the artifacts described below in [Configuration](/docs/api-gateway/configuration).
+
+<CodeBlockConfig hideClipboard filename="values.yaml">
+
+  ```yaml
+  apiGateway:
+  managedGatewayClass:
+  enabled: true
```

Comment:
Indentation is a little off here, but I'm not sure if we should keep this section. What is it intended to demonstrate? I tripped on the empty `managedGatewayClass`

```suggestion
  apiGateway:
    enabled: true
    managedGatewayClass:
```